src/chrome/client/doNotPauseWhileSteppingSession.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import * as utils from '../../utils';
+import { BaseWrappedSession } from './session';
+import { DebugProtocol } from 'vscode-debugprotocol';
+
+const steppingRequests = {
+    continue: true,
+    next: true,
+    stepIn: true,
+    stepOut: true,
+    pause: true,
+    restartFrame: true,
+};
+
+export class DoNotPauseWhileSteppingSession extends BaseWrappedSession {
```

Comment:
I don't quite understand why this is needed, if you are stepping quickly, don't you want to see the 'stopped' event ASAP, rather than waiting until stepping is done?

Reply:
This feature already existed on v1 here: https://github.com/Microsoft/vscode-chrome-debug-core/blob/ae8afa131a1dc37008d6a7df1869956ecc1c5d76/src/chrome/chromeDebugAdapter.ts#L731
I refactored/rewrote it so it's not mixed with the Pause logic.
I assumed that things broke if you sent a pause before sending a response to the step request that created the pause...

Reply:
I think pinezorro cares about that, vscode doesn't.